packages/lib/services/RevisionService.test.ts

```diff
@@ -619,6 +620,9 @@ describe('services/RevisionService', () => {
 		jest.advanceTimersByTime(100);
 
 		await Note.save({ id: note.id, title: 'test', body: 'StartA' });
+		// The first save after a revision collection will set the beforeChangeItemJson for subsequent saves to use. Wait for ItemChange.add to complete before subsequent
+		// saves, to avoid a race condition whereby the next save selects the beforeChangeItemJson before it has been saved and then uses a null value for subsequent saves
```

Comment:
Thanks for adding the comments but I think that is more information than needed since we already document what ItemChange.waitForAllSaved() is for, so please remove them